### DISCUSSION TITLE
error: can't find Rust compiler

### DISCUSSION THREAD

**Discussion by WaveBird** [PyO3/maturin#1547] [Q&A]
### Bug Description

My os is `Linux R7000-5545 2.6.36.4brcmarm #1 SMP PREEMPT Tue Sep 25 18:40:13 CST 2018 armv7l Merlin-Koolshare`
and I ran into this issue when I am installing the home assistant.

### Your maturin version (`maturin --version`)

None

### Your Python version (`python -V`)

3.10.7

### Your pip version (`pip -V`)

23.0.1

### What bindings you're using

bin

### Does `cargo build` work?

- [ ] Yes, it works

### If on windows, have you checked that you aren't accidentally using unix path (those with the forward slash `/`)?

- [ ] Yes

### Steps to Reproduce

python3 -m pip install maturin

**Comment by messense** [comment]
Please ask for help in `home-assistant` project instead, it needs to provide pre-compiled wheels for your platform.
